policy-test/tests/e2e_authorization_policy.rs

```diff
@@ -294,9 +296,19 @@ async fn network() {
             create_ready_pod(&client, web::pod(&ns))
         );
 
+        // Wait for the endpoints controller to populate the Endpoints resource.
+        let endpoints_ready = |obj: Option<&k8s::Endpoints>| -> bool {
+            if let Some(ep) = obj {
+                return ep.subsets.iter().flatten().count() > 0;
+            }
+            false
+        };
+        await_condition(&client, &ns, "web", endpoints_ready).await;
```

Comment:
nit/TIOLI: is there a reason `endpoints_ready` has to be repeated in all these tests, or can it be a helper in `linkerd_policy_test`?